src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests_ListPatterns.cs

```diff
@@ -1503,6 +1503,160 @@ public void M(int[] a)
                 );
         }
 
+        [ConditionalFact(typeof(CoreClrOnly))]
```

Comment:
:memo: This fails the verification in desktop and produces `Could not load file or assembly 'System.Runtime, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies.` when skipped.